frontend/components/Auction.vue

```diff
@@ -337,6 +350,9 @@ export default Vue.extend({
             }
             return null;
         },
+        isPageNetworkFake(): Boolean {
+            return this.$store?.getters['network/isPageNetworkFake'];
+        },
```

Comment:
Wouldn't this break the storybook? I would not use $store outside of containers or layouts

Instead you can use `auction.network` to check it against `FAKE_NETWORK_NAME`